GoogleAnalyticsImporter.php

```diff
@@ -359,4 +363,43 @@ private function getBoundsInTimezone(\Piwik\Period $period, $timezone)
 
         return [$date1, $date2];
     }
+
+    public function checkPendingImporters()
+    {
+        $notificationMessage = null;
+        if(!Common::getRequestVar('period', false) ||
+            !Common::getRequestVar('date', false)){
+            return;
+        }
+
+        $currentIdSite = Common::getRequestVar('idSite', -1);
+        if($currentIdSite === -1){
+            return;
+        }
+
+        $pending = false;
+        $instance = new ImportStatus();
+        try{
+            $status = $instance->getImportStatus($currentIdSite);
+            if ($status['status'] == ImportStatus::STATUS_ONGOING) {
+                $pending = true;
+            }
+        } catch (\Exception $exception){
+        }
+
+        if($pending === true){
+            $importStart = Date::factory($status['import_range_start']);
+            $importEnd = Date::factory($status['import_range_end']);
+            $startDate = Date::factory(Period\Factory::build(Common::getRequestVar('period'), Common::getRequestVar('date'))->getDateStart());
+            $endDate = Date::factory(Period\Factory::build(Common::getRequestVar('period'), Common::getRequestVar('date'))->getDateEnd());
+
+            if(($startDate >= $importStart || $startDate <= $importEnd) || ($endDate >= $importStart || $endDate <= $importEnd)){
+                $notificationMessage = 'Report data includes some Google Analytics historical data and is still being imported. The GA Data import should complete within a few days';
```

Comment:
Better to translate this by adding in the `lang/en.json` file.
Also do you think should we mention the import type like 
For GA -> The GA Data import should complete within a few days.
For GA4 -> The GA4 Data import should complete within a few days

Reply:
FYI, you have a key `isGA4` in the status to determine if its GA4 or not

Reply:
Thank @AltamashShaikh. This is done.